### ISSUE TITLE
Calculate TotalEstimatedJobs for the whole workflow

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#8785]
instead of calculating for the top level task only. I don't expect it to add too much load on the global workqueue, so we might publish this info instead of the current top level task only.

**Comment by ticoann** [comment]
I am assuming that we wants to get some estimate jobs for each tasks not without actually splitting the jobs. Are you considering merge/logcollect/cleanup job estimation as well?

**Comment by amaltaro** [comment]
Yes, I think we can still neglect any jobs that are not Production/Processing. Only estimates, not really splitting any work.
